unit_test/entities/controller/test_controller_class.py

```diff
@@ -14,22 +14,19 @@
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 """Test controller class."""
 import unittest
-from pathlib import Path
 from unittest.mock import Mock
-from datetime import datetime, timezone
+from datetime import datetime
 
-from omotes_simulator_core.adapter.transforms.mappers import EsdlControllerMapper
 from omotes_simulator_core.entities.assets.controller.controller_consumer import ControllerConsumer
 from omotes_simulator_core.entities.assets.controller.controller_producer import ControllerProducer
-from omotes_simulator_core.entities.esdl_object import EsdlObject
+from omotes_simulator_core.entities.assets.controller.controller_storage import ControllerStorage
 from omotes_simulator_core.entities.network_controller import NetworkController
 from omotes_simulator_core.entities.assets.asset_defaults import (
     PROPERTY_HEAT_DEMAND,
     PROPERTY_TEMPERATURE_RETURN,
     PROPERTY_TEMPERATURE_SUPPLY,
     PROPERTY_SET_PRESSURE,
 )
-from omotes_simulator_core.infrastructure.utils import pyesdl_from_file
 
 
 class ControllerTest(unittest.TestCase):
```

Comment:
You now extended this test to also include also storage. However now we lack a test without a storage, which should be included. Also you only add one storage it is better to also test for two storages